OpenRA.Mods.Cnc/Traits/Buildings/ClonesProducedUnits.cs

```diff
@@ -26,7 +26,7 @@ public class ClonesProducedUnitsInfo : ConditionalTraitInfo, Requires<Production
 
 		[FieldLoader.Require]
 		[Desc("e.g. Infantry, Vehicles, Aircraft, Buildings")]
-		public readonly string ProductionType = "";
+		public readonly string ProductionType = null;
```

Comment:
```suggestion
		public readonly string ProductionType;
```

Couldn't we just do this instead? It's null by default